grr/server/grr_response_server/bin/grrafana.py

```diff
@@ -155,6 +162,13 @@ def _FetchDatapointsForTargets(
   return response
 
 
+def _GrafanaTimestampToTimestampObj(
```

Comment:
Given that we explicitly test this function in the testing module, I'd suggest not making it module-private (i.e. removing the leading "_").